cmd/kops/get.go

```diff
@@ -87,6 +114,102 @@ func NewCmdGet(f *util.Factory, out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func RunGet(context Factory, out io.Writer, options *GetOptions) error {
+
+	client, err := context.Clientset()
+	if err != nil {
+		return err
+	}
+
+	cluster, err := client.Clusters().Get(options.clusterName)
+	if err != nil {
+		return err
+	}
+
+	if cluster == nil {
+		fmt.Fprintf(os.Stderr, "No cluster found\n")
+		return nil
+	}
+
+	clusterList := &api.ClusterList{}
+	clusterList.Items = make([]api.Cluster, 1)
+	clusterList.Items[0] = *cluster
+
+	args := make([]string, 0)
+
+	clusters, err := buildClusters(args, clusterList)
+
+	ig, err := client.InstanceGroups(options.clusterName).List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	if ig == nil || ig.Items == nil || len(ig.Items) == 0 {
+		fmt.Fprintf(os.Stderr, "No instance groups found\n")
+	}
+
+	instancegroups, err := buildInstanceGroups(args, ig)
+	if err != nil {
+		return err
+	}
+
+	switch options.output {
+	case OutputYaml:
+
+		err = clusterOutputYAML(clusters, out)
+		if err != nil {
+			return err
+		}
+
+		if err := writeYAMLSep(out); err != nil {
+			return err
+		}
+
+		err = igOutputYAML(instancegroups, out)
+		if err != nil {
+			return err
+		}
+
+	case OutputJSON:
+		return fmt.Errorf("not implemented")
```

Comment:
Let's figure this out because json is important and we should be able to get there.

It's not valid json because we're not wrapping the output properly to be json. 

What if we did something like this: 

```json
{
 "clusterSpec": { here is output from clusterOutputJson },
 "instanceGroups": [
  { master-1 },
  { master-2 },
  { master-3 },
  { nodes }
 ] 
}
```

It would be trivial to rework the output to get to the output above. Side note: Can we read something like I described above on `kops create -f`? If not, I think we should be able to.